static/js/index.js

```diff
@@ -216,6 +216,12 @@
   var renderHits = function(search, result) {
 
     var html = "";
+
+    if (result.ResultCount) {
```

Comment:
This should be result["c"] now? (and not result.c because of the closure compiler optimizations!)